dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapAdapter.java

```diff
@@ -310,7 +328,16 @@ protected void setSSLSupport(SSLSupportOptional sslSupport) {}
      */
     @Override
     public Root get(Root root) throws WIMException {
-        final String METHODNAME = "get";
+        getConfigReadLock();
```

Comment:
Do you want to have the entry/exit trace on these new wrapper methods? Or add `@Trivial` ?

Reply:
I do, simply to determine whether it plays a role in any performance issues when debugging.